src/main/java/appeng/client/render/crafting/MolecularAssemblerRenderer.java

```diff
@@ -56,7 +58,7 @@
 
     public static final ResourceLocation LIGHTS_MODEL = AppEng.makeId("block/molecular_assembler_lights");
 
-    private static final RenderType MC_161917_RENDERTYPE_FIX = createRenderType();
+    private static final RenderType MC_161917_RENDERTYPE_FIX = RenderTypeAccess.createRenderType();
```

Comment:
This isn't some fancy hack to replace an AT. Just a custom rendertype for the molecular assembler (or maybe a workaround type for the vanilla bug)

Reply:
Well no, it's fixing a bug around alpha testing in the vanilla render type

Reply:
The related vanilla issue is marked as resolved, so this might be the intentional behaviour for minecraft. For whatever reason.
If we are not happy with the defaults and want a different rendering behaviour, then it is no longer a workaround. Just our custom solution.

Reply:
The behavior between fabulous and fancy rendering is different in vanilla

Reply:
Which may or may not be intentional. Regardless we want a different behaviour than the defaults provided by either. And it should be always the same regardless of the setting.